tests/test_printer.py

```diff
@@ -761,6 +763,28 @@ def test_densearray_attr():
     assert_print_op(parsed, prog, None)
 
 
+def test_float_attr_specials():
```

Comment:
Do these also parse correctly?

Reply:
Absolutely not 🫠 Working on it though

Reply:
We currently can't parse special floats at all